src/Components/Publishing/Byline/ClassicByline.tsx

```diff
@@ -12,33 +16,55 @@ interface ClassicBylineProps {
 
 export const ClassicByline: React.SFC<ClassicBylineProps> = props => {
   const {
-    article: { contributing_authors, author, published_at },
+    article: {
+      channel,
+      contributing_authors,
+      author,
+      published_at,
+      slug,
+      social_title,
+      thumbnail_title,
+    },
     date,
   } = props
 
   const contributors = getAuthorByline(contributing_authors, false)
```

Comment:
There's a good amount of logic in this component. I see that there are snapshot tests - do they cover all the branches? If not, now is probably a good time to add coverage.